src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1243,6 +1260,54 @@ typedef struct {
                      //!< RTCDataChannel object with the same id as the other peer.
 } RtcDataChannelInit, *PRtcDataChannelInit;
 
+/////////////////////////////////////////////////////
+/// Metrics/Stats Related structures
+/////////////////////////////////////////////////////
+
+/**
+ * @brief Collection of ICE related stats
+ * Reference: https://www.w3.org/TR/webrtc-stats/#ice-server-dict*
+ * Reference: https://www.w3.org/TR/webrtc-stats/#icecandidate-dict*
+ * Reference: https://www.w3.org/TR/webrtc-stats/#candidatepair-dict*
+ */
+typedef struct {
+    UINT32 version; //!< Structure version
+    RtcIceServerStats rtcIceServerStats; //!< Server related stats. Reference in Stats.h
+    RtcIceCandidateStats rtcIceCandidateStats; //!< Single candidate stats. Reference in Stats.h
+    RtcIceCandidatePairStats rtcIceCandidatePairStats; //!< Candidate pair stats. Reference in Stats.h
+} RtcIceMetrics, *PRtcIceMetrics;
+
+/**
+ * @brief Collection of RTP stream related stats
+ * Reference: https://www.w3.org/TR/webrtc-stats/#remoteinboundrtpstats-dict*
+ * Reference: https://www.w3.org/TR/webrtc-stats/#outboundrtpstats-dict*
+ * Reference: https://www.w3.org/TR/webrtc-stats/#transportstats-dict*
+ */
+typedef struct {
+    UINT32 version; //!< Structure version
+    RtcRemoteInboundRtpStreamStats rtcInboundStats; //!< Inbound RTP Stats. Reference in Stats.h
+    RtcOutboundRtpStreamStats rtcOutboundStats; //!< Outbound RTP Stats. Reference in Stats.h
+    RtcTransportStats rtcTransportStats; //!< Transport stats. Reference in Stats.h
+} RtcStreamMetrics, *PRtcStreamMetrics;
+
+/**
+ * @brief SignalingStats Collection of signaling related stats. Can be expanded in the future
+ */
+typedef struct {
+    UINT32 version; //!< Structure version
+    SignalingClientStats signalingClientMetrics; //!< Signaling client metrics stats. Reference in Stats.h
```

Comment:
btw, should the name be then signalingClientStats?

Reply:
Just having it consistent with the Stats suffix for other metrics, thats all.